### DISCUSSION TITLE
architecture/container

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#192] [General]
# architecture/container

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/architecture/container.html

**Comment by Itswag** [comment]
CNI（~~Container Runtime Interface~~，容器网络接口规范） Container Network Interface

**Reply by isno** [reply]
谢谢，这儿写错了，已修改。

**Comment by CowSmiles** [comment]
“Docker 把与内部责管理容器执行、分发、监控、网络、构建、日志等功能的模块重构为 containerd 项目 。”这里应该少了一个字，“内部负责”

**Reply by isno** [reply]
谢谢，已修改。

**Comment by liusifanshui** [comment]
"所以，早期的 PaaS 平台探索的技术并没有形成大的行业趋势，只局限在一些的特定的领域"，建议把“一些的特定的领域”改为“一些特定的领域”
